supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
 *Incorrect forms*:
 
 *See also*: xref:in-place-upgrade[in-place upgrade], xref:clean-install[clean install]
+
+*NEWLY ADDED*:
+[discrete]
+[[active-directory-forest]]
+==== image:images/yes.png[yes] Active Directory forest (noun)
+*Description*: An Active Directory (AD) forest is a set of one or more domain trees which share a common global catalog, directory schema, logical structure, and directory configuration. The forest represents the security boundary within which users, computers, groups, and other objects are accessible.
+For more information, see the Microsoft document on link:https://docs.microsoft.com/en-us/windows/win32/ad/forests[Forests].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[active-directory-global-catalog]]
+==== image:images/yes.png[yes] Active Directory global catalog (noun)
+*Description*: The global catalog is a feature of Active Directory (AD) that allows a domain controller to provide information on any object in the forest, regardless of whether the object is a member of the domain controller’s domain. Domain controllers with the global catalog feature enabled are referred to as global catalog servers.  The global catalog provides a searchable catalog of all objects in every domain in a multi-domain Active Directory Domain Services (AD DS).
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:active-directory-forest[active-directory-forest]
```

Comment:
Can you change the "active-directory-forest" in the brackets to be the correct display name of "Active Directory forest"?

Same for all other xrefs that you added

Reply:
Oh, I see now. Thanks for catching this. Will correct all of them.